app/main.py

```diff
@@ -1,25 +1,91 @@
-from abc import ABC
+from __future__ import annotations
+from abc import ABC, abstractmethod
+
+
+from typing import Any, Callable
 
 
 class IntegerRange:
-    pass
+    def __init__(self, min_amount: int, max_amount: int) -> None:
+        self.min_amount = min_amount
+        self.max_amount = max_amount
+        self.name = None
+
+    def __get__(self, instance: Visitor, owner: Any) -> dict:
+        return instance.__dict__[self.name]
+
+    def __set__(self, instance: Visitor, value: int) -> None:
+        if not (self.min_amount <= value <= self.max_amount):
+            raise ValueError(
+                f"Value must be between {self.min_amount} "
+                f"and {self.max_amount}"
+            )
+        instance.__dict__[self.name] = value
+
+    def __set_name__(self, owner: Any, name: str) -> None:
+        self.name = name
 
 
 class Visitor:
-   pass
+    age = IntegerRange(0, 150)
+    weight = IntegerRange(0, 300)
+    height = IntegerRange(0, 300)
+
```

Comment:
Why do you need these?

Reply:
I don't know to be honest, I just decided to complete a task that I didn't solve before and this part of the code was done for me a few months ago)